### DISCUSSION TITLE
Upgrading

### DISCUSSION THREAD

**Discussion by quasiblob** [bmaltais/kohya_ss#220] [Q&A] [answered]
I see that the main page of repo says this about upgrading:

"When a new release comes out, you can upgrade your repo with the following commands in the root directory:
git pull
.\venv\Scripts\activate
pip install --use-pep517 --upgrade -r requirements.txt"

There's also upgrade.ps1, which I've run once accidentally. 

It did seem to fetch some updates, but what is the difference?

I didn't yet have time to check what the ps1 file contains, neither am I a pip package manager user.

**Comment by bmaltais** [comment, marked as answer]
Essentially doing the same but in an easy to run script ;-) No harm done.
